tomcat/available-versions

```diff
@@ -0,0 +1 @@
+9.0.76
```

Comment:
here you should list all the versions. For major version 9, you can get the list from here:
https://archive.apache.org/dist/tomcat/tomcat-9/ but you can omit the milestones (`9.0.0.M*`).